src/lib/webpack/run-webpack.js

```diff
@@ -38,7 +38,12 @@ const devBuild = async (env, onprogress) => {
 				let serverAddr = `${protocol}://${host}:${chalk.bold(port)}`;
 				let localIpAddr = `${protocol}://${ip.address()}:${chalk.bold(port)}`;
 
-				process.stdout.write(chalk.green('\nCompiled successfully!!\n\n'));
+				if (stats.hasErrors()) {
+					process.stdout.write(chalk.red('\Build failed!\n\n'));
+				} else {
+					process.stdout.write(chalk.green('\nCompiled successfully!\n\n'));
+				}
```

Comment:
Better to expose the IP addresses & port message only when the build is successful !!

```javascript
    if (userPort !== port) {
       process.stdout.write(`Port ${chalk.bold(userPort)} is in use, using ${chalk.bold(port)} instead\n\n`);
    }
    process.stdout.write('You can view the application in browser.\n\n');
    process.stdout.write(`${chalk.bold('Local:')}            ${serverAddr}\n`);
    process.stdout.write(`${chalk.bold('On Your Network:')}  ${localIpAddr}\n`);
```